Scripts/build_and_install.lua

```diff
@@ -0,0 +1,7 @@
+import("core.base.option")
+import("core.base.task")
+
+function main()
+    task.run("build", option.get("name"), "-y")
```

Comment:
Doesn't seem like the `-y` option is actually passed in because it still prompts me.